src/services/meteors-service.js

```diff
@@ -0,0 +1,53 @@
+const axios = require("axios");
```

Comment:
Check please all usages of `''` and `""` - it's better not to mix them up

Reply:
prettier can help with this
Just add some lines to `.prettierrc`

```
  "singleQuote": true
```

Reply:
There is no such file on this repo :)

Reply:
yep, I will add prettier to my project, Thank you!